### PR TITLE
Pcs stata

### PR DESCRIPTION
**Pull request by paulasanematsu**
- Moved IQSS Stata example to Languages/Stata
- Added multi-core example
- Updated existing example
- Updated README accordingly

### CODE REVIEW

File: Languages/STATA/README.md [hunk 1]
```diff
@@ -18,142 +67,223 @@ describe
...
+#SBATCH -t 0-00:30            # time in D-HH:MM
+#SBATCH -N 1                  # number of nodes
+#SBATCH -c 4                  # number of cores
+#SBATCH --mem=4000            # total memory in MB
+
+# Load required modules
+module load stata/17.0-fasrc01
+
+# Run program
+stata-mp -b hello_mp.do
```
Comment (pedmon) [inline Languages/STATA/README.md:237]: This should be prefixed by srun per: https://docs.rc.fas.harvard.edu/kb/running-jobs/#Using_Threads_such_as_OpenMP

File: Languages/STATA/run_hello_mp.sbatch [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+#SBATCH -t 0-00:30            # time in D-HH:MM
+#SBATCH -N 1                  # number of nodes
+#SBATCH -c 4                  # number of cores
+#SBATCH --mem=4000            # total memory in MB
+
+# Load required modules
+module load stata/17.0-fasrc01
+
+# Run program
+stata-mp -b hello_mp.do
```
[the remark this answers is not in the document]
Comment (paulasanematsu) [reply Languages/STATA/run_hello_mp.sbatch:15]: Both with and without `srun -c $SLURM_CPUS_PER_TASK` work. However, I haven't tested with a "real" code to be sure. Most examples on other HPC centers don't use srun. I wonder if it makes any difference. Either way, I can add the srun to be safe.

[the remark this answers is not in the document]
Comment (pedmon) [reply Languages/STATA/run_hello_mp.sbatch]: Thanks. I've seen it make a difference the past. Without the srun slurm makes assumptions about how many cores to give a process. Usually its fine but some times it doesn't do the right thing. Strictly speaking one should always include the srun -c but you can get away without it in most cases.

**Review by pedmon — APPROVED** [review]

File: Languages/STATA/run_hello_mp.sbatch (hunk 2 above)
[the remark this answers is not in the document]
Comment (paulasanematsu) [reply Languages/STATA/run_hello_mp.sbatch]: That's good to know it's that slurm may give a different number of cores for the process. Thanks for the clarification.

**Pull request merged**